templates/UtilFile.stencil

```diff
@@ -0,0 +1,7 @@
+import Foundation
+ 
+ public func formatDateToString(_ date: Date) -> String {
+     let dateFormatter = DateFormatter()
+     dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
+     return dateFormatter.string(from: date)
+ }
```

Comment:
Does this not work?

```Swift
extension Date: LosslessStringConvertible {
    public init?(_ description: String) {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
        self = dateFormatters.date(from: description) ?? Date()
    }
}
```

Reply:
Yeah, nevermind this LosslessString stuff. Just use AzureCore's 
`String(describing: Date, withFormat: Date.Format)` and `Date(_: String?, format: Date.Format)` methods.